nodejs/nodejs-guestbook/src/backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use base node 8 image from Docker hub
-FROM node:8
+FROM node:12-alpine
```

Comment:
Is there a particular reason why we should use `node:12-alpine`? Most of the other GCP Node.js samples are using `node:12-slim`, so it might be preferable to use the same image for consistency.

Reply:
Good call!